doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -509,6 +509,10 @@ This section provides detailed lists of changes by :ref:`script <scripts>`.
 
     * :kconfig:option:`CONFIG_PM_PARTITION_ALIGN_SETTINGS_STORAGE` Kconfig option to specify the alignment of the settings storage partition.
 
+  * Fixed:
+
+    * An issue with the driver and devicetree symbol for external flash. The driver was sometimes NULL even if the DT node was chosen.
```

Comment:
```suggestion
   * Fixed an issue with the driver and devicetree symbol for the external flash memory where the driver was sometimes NULL, even if the DT node was chosen.
```

No need to create a sublist for fixed if the item in the list is just 1

Reply:
The heading of the above list is "Added:". Should I change it to "Changed:" and add "Added ..." to the previous point?

Reply:
See https://github.com/nrfconnect/sdk-nrf/pull/9169/commits/94f4034a9ad0d1042a47027d0ee05fbf7fea5346

Reply:
Yes, the change looks good, at least till there will be just 1 item for each of the categories (added and fixed)